pyproject.toml

```diff
@@ -17,7 +17,7 @@ profile = "black"
 skip_gitignore = true
 
 [tool.mypy]
-python_version = "3.10"
+python_version = "3.11"
```

Comment:
3.12?
I think the Circle CI config used 3.12, the ping_server Dockerfile used 3.10.